src/praveenReddyDec19Assignment3/SumarrayQ10.java

```diff
@@ -0,0 +1,34 @@
+package praveenReddyDec19Assignment3;
+/*
+ WAP to sum of two array 
+ a. firstArray :- {1, 5, 9, 3, 7} 
+ b. secondAraay:- {1, 7,  9, 3} 
+ c. output :- 4
+ */
+
+public class SumarrayQ10 {
+	static void SumOf2Arrays(int a[], int b[])
```

Comment:
for this question, we have to create one method which takes int array and returns a sum of an element.
we have to call this method two times.